vmdb/app/controllers/application_controller/ci_processing.rb

```diff
@@ -1844,4 +1846,7 @@ def owner_changed?(owner)
     @edit[:new][owner] != @edit[:current][owner]
   end
 
+  def update_breadcrumb_trail(trail_end)
```

Comment:
This method is confusing upon reading.  When you read the caller, it seems like you are going to "update the breadcrumb trail with the following parameter", but that's not what it does at all.  Perhaps `pop_breadcrumb_trail` might be a better method name?